FirebaseAuth/Sources/Backend/RPC/FIRGetOOBConfirmationCodeRequest.m

```diff
@@ -280,10 +295,18 @@ - (nullable id)unencodedHTTPRequestBodyWithError:(NSError *_Nullable *_Nullable)
   if (_handleCodeInApp) {
     body[kCanHandleCodeInAppKey] = @YES;
   }
-
```

Comment:
super-nit: restore this newline and add newlines between if blocks below to be consistent with the rest of the code here?